docs/reference-guides/block-api/block-metadata.md

```diff
@@ -497,6 +497,23 @@ _Note: In JavaScript you can provide a function for the `isActive` property, and
 
 See the [the variations documentation](/docs/reference-guides/block-api/block-variations.md) for more details.
 
+### Block Hooks
+
+-   Type: `object`
+-   Optional
+-   Property: `blockHooks`
+-   Since: `WordPress 6.4.0`
+
+```json
+{
+	"blockHooks": {
+		"my-plugin/banner": "after"
+	}
+}
+```
+
+Block Hooks is the API that allows a block to hook into the rendering of another block. It will enable a block to render its content before or after another block. It's also possible to hook into the rendering of a parent block and prepend or append the block to the list of child blocks. The key is the name of the block to hook into, and the value is the position to hook into. Take a look at the [Block Hooks documentation](/docs/reference-guides/block-api/block-registration.md#block-hooks-optional) for more info about the available configurations.
```

Comment:
This paragraph mentions _rendering_ of hooked blocks; maybe it's worth mentioning that they aren't only rendered (on the frontend) but also available for customization in the editor 🤔 

(Not a blocker; we can polish documentation later.)

Reply:
Something like

```suggestion
Block Hooks is an API that allows a block to automatically insert itself next to all instances of a given block type, in a relative position also specified by the "hooked" block. That is, a block can opt to be inserted before or after a given block type, or as its first or last child (i.e. to be prepended or appended to the list of its child blocks, respectively). Hooked blocks will appear both on the frontend and in the editor (to allow for customization by the user).

The key is the name of the block to hook into, and the value is the position to hook into. Take a look at the [Block Hooks documentation](/docs/reference-guides/block-api/block-registration.md#block-hooks-optional) for more info about available configurations.
```

(But again, we can address this later.)